drivers/sensor/tdk/icp201xx/icp201xx_drv.c

```diff
@@ -217,7 +217,7 @@ static int icp201xx_channel_get(const struct device *dev, enum sensor_channel ch
 	struct icp201xx_data *data = (struct icp201xx_data *)dev->data;
 
 	if (!(chan == SENSOR_CHAN_AMBIENT_TEMP || chan == SENSOR_CHAN_PRESS ||
-	      chan == SENSOR_CHAN_ALTITUDE)) {
+	      (chan == SENSOR_CHAN_ALTITUDE && IS_ENABLED(CONFIG_FPU)))) {
```

Comment:
Shouldn't this be:
```suggestion
	      (chan == SENSOR_CHAN_ALTITUDE && !IS_ENABLED(CONFIG_FPU)))) {
```

Would be nice if the driver had a Kconfig symbol to enable/disable the altitude channel and that symbol depend on the FPU being enabled. But I think that's beyond the scope of this PR.